### ISSUE TITLE
 Error failed to bundle project: error running appimage.sh

### ISSUE THREAD

**Issue by LeonBirkel** [tauri-apps/tauri-action#905]
Hey,
I ccan build normally on my local machine, but have problemts with tauri-actions. 
I get this error (verbose logs in the txt file)
Can anybody help me with that? 

Thanks you in advance!

[0_publish-tauri (ubuntu-22.04).txt](https://github.com/user-attachments/files/16919360/0_publish-tauri.ubuntu-22.04.txt)

**Comment by FabianLars** [comment]
Please re-run it with `args: --verbose` set on the tauri-action step so we actually see why it fails.

**Comment by LeonBirkel** [comment]
Thanks for the fast reply :)
I did that, maybe at the wrong position? 

This is my tauri-action:
```name: "publish"

on:
  push:
    branches:
      - production

# This is the example from the readme.
# On each push to the `release` branch it will create or update a GitHub release, build your app, and upload the artifacts to the release.

jobs:
  publish-tauri:
    permissions:
      contents: write
    strategy:
      fail-fast: false
      matrix:
        include:
          # - platform: "macos-latest" # for Arm based macs (M1 and above).
          #   args: "--target aarch64-apple-darwin"
          # - platform: "macos-latest" # for Intel based macs.
          #   args: "--target x86_64-apple-darwin"
          - platform: "ubuntu-22.04" # for Tauri v1 you could replace this with ubuntu-20.04. # ubuntu-22.04
            args: ""
          # - platform: "windows-latest"
          #   args: ""

    runs-on: ${{ matrix.platform }}
    steps:
      - uses: actions/checkout@v4

      - name: setup node
        uses: actions/setup-node@v4
        with:
          node-version: lts/*

      - name: install Rust stable
        uses: dtolnay/rust-toolchain@stable
        with:
          # Those targets are only used on macos runners so it's in an `if` to slightly speed up windows and linux builds.
          targets: ${{ matrix.platform == 'macos-latest' && 'aarch64-apple-darwin,x86_64-apple-darwin' || '' }}

      - name: install dependencies (ubuntu only)
        if: matrix.platform == 'ubuntu-22.04' # This must match the platform value defined above.
        run: |
          sudo apt-get update
          sudo apt-get install -y libwebkit2gtk-4.0-dev libwebkit2gtk-4.1-dev libappindicator3-dev librsvg2-dev patchelf
        # libwebkit2gtk-4.1-dev
        # webkitgtk 4.0 is for Tauri v1 - webkitgtk 4.1 is for Tauri v2.
        # You can remove the one that doesn't apply to your app to speed up the workflow a bit.

      - name: install frontend dependencies
        run: npm install # change this to npm, pnpm or bun depending on which one you use.

      - uses: tauri-apps/tauri-action@v0
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          TAURI_PRIVATE_KEY:  ${{ secrets.BASTION_KEY_PRIVATE }}
          TAURI_KEY_PASSWORD: ${{ secrets.BASTION_KEY_PASSWORD }} 
          SIGN: "1"
          SIGN_KEY: ${{ secrets.BASTION_SIGNING_KEY_PRIVATE }} 
          APPIMAGETOOL_SIGN_PASSPHRASE: ${{ secrets.APPIMAGETOOL_SIGN_PASSPHRASE }} 
          releaseId: v__VERSION__
          tagName: bastion-v__VERSION__ # the action automatically replaces \_\_VERSION\_\_ with the app version.
          releaseName: "BASTION v__VERSION__"
          releaseBody: "See the assets to download this version and install."
          releaseDraft: true
          prerelease: false
          args: ${{ matrix.args }} --verbose
  notify-webserver:
    runs-on: ubuntu-latest
    needs: publish-tauri  # Run only after the Tauri build is successful
    steps:
      - name: Trigger web server update (example)
        run: |
          curl -X POST https://bastion-software.com/api/app/version/new_release \
            -H "Content-Type: application/json" \
            -d '{"version": "${{ steps.publish-tauri.outputs.release_tag }}"}'```

**Comment by LeonBirkel** [comment]
Ok, I fixed the issue, my SIGN_KEY fo ubuntu seemed to be somehow corrupted. 
Thanks for you help anyways. I encountered another problem. But will open another Issue for it. Since it is unrelated
